### ISSUE TITLE
Cron is broken

### ISSUE THREAD

**Issue by Silox** [ZeusWPI/Gandalf#509]
```
/usr/local/rvm/gems/ruby-2.2.0@global/gems/bundler-1.7.12/lib/bundler/definition.rb:22:in `build': /home/gandalf/production/releases/20141217221558/Gemfile not found (Bundler::GemfileNotFound)
	from /usr/local/rvm/gems/ruby-2.2.0@global/gems/bundler-1.7.12/lib/bundler.rb:155:in `definition'
	from /usr/local/rvm/gems/ruby-2.2.0@global/gems/bundler-1.7.12/lib/bundler.rb:118:in `setup'
	from /usr/local/rvm/gems/ruby-2.2.0@global/gems/bundler-1.7.12/lib/bundler/setup.rb:17:in `<top (required)>'
	from /usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `require'
	from /usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:128:in `rescue in require'
	from /usr/local/rvm/rubies/ruby-2.2.0/lib/ruby/2.2.0/rubygems/core_ext/kernel_require.rb:39:in `require'
	from bin/rails:14:in `<main>'
```

**Comment by Silox** [comment]
See for the fix https://github.com/javan/whenever/issues/555

**Comment by Silox** [comment]
Fixed in 91e23e36366677a752d6adaeda390a7b340476c6
